tests/unwind_stack.rs

```diff
@@ -50,9 +50,9 @@ fn unwind_stack() -> Result<(), Box<dyn std::error::Error>> {
 
     let expected = &[
         "breakpoint",
-        "_ZN5hello4main17h",
-        "_ZN3std2rt10lang_start28_$u7b$$u7b$closure$u7d$$u7d$17h",
-        "_ZN3std2rt19lang_start_internal17h",
+        "hello::main::",
+        "std::rt::lang_start::{{closure}}::",
+        "std::rt::lang_start_internal::",
```

Comment:
I think `get_address_symbol_name` should not demangle the name. Instead there could be an extra method `get_address_demangled_name`.